python/ray/rllib/evolution_strategies/evolution_strategies.py

```diff
@@ -30,8 +30,8 @@
 DEFAULT_CONFIG = dict(
     l2coeff=0.005,
     noise_stdev=0.02,
-    episodes_per_batch=10000,
-    timesteps_per_batch=100000,
+    episodes_per_batch=1000,
+    timesteps_per_batch=10000,
```

Comment:
Made this a bit lower since it still seems to work and 10k is too high on smaller machines. Though perhaps it would work better with 10k